lib/mongo/server/app_metadata.rb

```diff
@@ -109,25 +109,31 @@ def initialize(options = {})
       # @api private
       attr_reader :purpose
 
+      # @return [ Hash | nil ] Information server api.
+      #
+      #   Thes hash can have the following items:
+      #   - *:version* -- string
+      #   - *:strict* -- boolean
+      #   - *:deprecation_errors* -- boolean
+      #
+      # @api private
+      attr_reader :server_api
+
       # @return [ Array<Hash> | nil ] Information about libraries wrapping
       #   the driver.
       attr_reader :wrapping_libraries
 
-      # Get the bytes of the ismaster message including this metadata.
+      # Get the metadata as BSON::Document to be sent to
+      # a server for handshake purposes. The document is incomplete, and should
```

Comment:
s/for handshake purposes/as part of the handshake/.

I also wouldn't say that the document is incomplete because that is confusing - *all* of the metadata is included, yes? I think just leaving "this metadata document must be incorporated into a suitable handshake command" would be sufficient.

Reply:
Good point, corrected accordingly.